packages/paste-website/src/pages/components/tabs/index.mdx

```diff
@@ -85,20 +86,43 @@ export const getStaticProps = async () => {
+
+Having a distinction between the two components (Tabs and In Page Nav) leads to an improved user experience by visually communicating that interacting will lead to different behaviors. It's also important for accessibility to select the correct component to use based on your use case, as using the correct component gives users of assistive technologies a better idea of where the interaction will lead.
+
+Choose Tabs when users don’t need to see content from multiple views simultaneously. When thinking through what type of content to put in your Tabs, remember that the content should be parallel across the board. What lives in one Tab should be a different view of what lives in another Tab.
+
+_Source: [Nielsen Norman Group](https://www.nngroup.com/articles/tabs-used-right/)_
+
+The Tabs component contains the following elements:
+
+- Tabset: a composition of a Tablist and Tabpanels
+- Tablist: a collection of Tabs
+- Tab: clickable element that causes a selection within the Tabset
+- Tabpanel: contains the content associated with the selected Tab
 
 ### Accessibility
 
+Tabs should never contain interactive elements, as doing so makes it difficult for users of assistive technologies to focus or click on the Tabs. Interactive elements may live within the Tab Panels.
+
 When the user is focused on a tab in a TabList, the following keyboard interactions apply:
 
-- Right and left arrow move the user between tabs and opens the currently focused tab's panel
-  - If the tabs are vertical, up and down arrows move between tabs instead of right and left
-  - If the currently focused tab is disabled, the tab panel does not open
+- Horizontal Tabs: `Right` and `Left` arrow keys move the user between Tabs and open the currently focused Tab's panel
+- Vertical Tabs: `Up` and `Down` arrow keys move between Tabs and open the currently focused Tab's panel
+- If the currently focused Tab is disabled, the Tab panel does not open
+
+### Tabs vs. In Page Navigation
+
+Tabs are used to layer related pieces of information together and display one layer at a time on the same URL. Use Tabs to alternate between views within the same context. [In Page Navigation](/components/in-page-navigation) is a collection of anchors, rather than buttons. Use In Page Navigation to switch between different, related pages. Tabs replace the entire view based on the selected tab. In Page Navigation links navigate the user to a new page.
 
 ## Examples
 
 ### Horizontal Tabs
 
+Use Horizontal Tabs in most cases, as they most realistically mimic a physical tab, and they catch the user’s eye because they live directly above the content.
```

Comment:
```suggestion
Use Horizontal Tabs in most cases, as they most realistically mimic a physical tab, and they catch the user’s attention because they live directly above the content.
```